tests/integration/test_reload_zookeeper/test.py

```diff
@@ -8,7 +8,7 @@
 
 
 cluster = ClickHouseCluster(__file__, zookeeper_config_path='configs/zookeeper.xml')
-node = cluster.add_instance('node', with_zookeeper=True)
+node = cluster.add_instance('node', with_zookeeper=True, user_configs=["configs/users.xml"])
```

Comment:
Why do we need this change?

Reply:
It seems unnecessary to me.

Reply:
Ok, thanks.

Reply:
I understood, we need it to have an exception from readonly table.

Reply:
It's used to enable the required configuration in `configs/users.xml`